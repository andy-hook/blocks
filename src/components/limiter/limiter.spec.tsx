import React from "react"
import { render } from "@test-utils"
import Limiter from "./limiter"

describe("<Limiter />", () => {
  it("renders correctly", () => {
    const tree = render(<Limiter>Child text</Limiter>)
    expect(tree).toBeTruthy()
  })
})
