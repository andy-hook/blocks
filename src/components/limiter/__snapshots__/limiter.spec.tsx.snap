// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Limiter /> renders as large 1`] = `
<div
  className="sc-htpNat dzIMZJ"
>
  Child text
</div>
`;

exports[`<Limiter /> renders as medium 1`] = `
<div
  className="sc-bwzfXH hVRByH"
>
  Child text
</div>
`;

exports[`<Limiter /> renders as small 1`] = `
<div
  className="sc-bdVaJa kZPNX"
>
  Child text
</div>
`;

exports[`<Limiter /> renders correctly 1`] = `
<div
  className="sc-bwzfXH hVRByH"
>
  Child text
</div>
`;
